tests/test_multistore_file.py

```diff
@@ -0,0 +1,44 @@
+"""Unit tests for oauth2client.multistore_file."""
+
+import errno
+import os
+import tempfile
+import unittest
+
+from oauth2client import multistore_file
+
+
+class MultistoreFileTests(unittest.TestCase):
+
+    def test_lock_file_raises_ioerror(self):
+
+        filehandle, filename = tempfile.mkstemp()
+        os.close(filehandle)
+
+        try:
+            for error_code in (errno.EDEADLK, errno.ENOSYS, errno.ENOLCK):
+
+                class MockLockedFile(object):
```

Comment:
Please place the class at module scope. It's not that the class needs a reason to move to module-scope, it's that module-scope is the default place for classes and they should only be promoted into the scope of other code elements when necessary (and "is only ever used from" is not sufficient reason).